bin/get-cutouts

```diff
@@ -0,0 +1,386 @@
+#!/usr/bin/env python3
+
+"""MPI wrapper to get a large number of image cutouts. Only run ahead of a VAC
+release.
+
+get-cutouts --survey cmx --mp 16 --dry-run
+
+shifterimg pull dstndstn/viewer-cutouts:latest
+shifter --image dstndstn/viewer-cutouts cutout --output cutout.jpg --ra 234.2915 --dec 16.7684 --size 256 --layer ls-dr9 --pixscale 0.262 --force
+
+shifter --image dstndstn/viewer-cutouts bash
+/global/u2/i/ioannis/code/desihub/fastspecfit/bin/get-cutouts --survey cmx --program other --mp 128
+
+"""
+import pdb # for debugging
+
+import os, sys, time
+import numpy as np
+import fitsio
+import multiprocessing
+from glob import glob
+
+def _cutout_one(args):
+    return cutout_one(*args)
+
+def cutout_one(jpegfile, ra, dec, dry_run):
+    """
+    pixscale = 0.262
+    width = int(30 / pixscale)   # =114
+    height = int(width / 1.3) # =87 [3:2 aspect ratio]
+
+    """
+    import subprocess
+
+    cmd = 'cutout --output {} --ra {} --dec {} --width 114 --height 87 --layer ls-dr9 --pixscale 0.262 --force'.format(
+        jpegfile, ra, dec)
+    if dry_run:
+        print(cmd)
+    else:
+        subprocess.call(cmd.split())    
```

Comment:
MPI ranks forking system calls is fragile and vendor dependent and in particular known to not work robustly on Perlmutter.  Is there possibly a library version of `cutout` where you could directly call a python function instead of spawning a system call?

This kind of system call will usually work, even on Perlmutter, but some small fraction of the time it will fail in obscure ways and can create a big headache.  Beware.  Note that this is different from Cori where it did work.  NERSC staff did a lot of work with HPE trying to get it to also work on Perlmutter and eventually gave up.